sonarqube/assets/dashboards/overview.json

```diff
@@ -0,0 +1,762 @@
+{
+    "author_name": "Datadog",
+    "description": "",
+    "layout_type": "ordered",
+    "template_variables": [],
+    "title": "Sonarqube Overview",
+    "widgets": [
+        {
+            "definition": {
+                "background_color": "white",
+                "content": "[SonarQube](https://www.datadoghq.com/blog/datadog-sonarqube-integration) is an open-source tool for static code analysis that integrates with your existing CI pipeline to run regular code checks on your codebase.\n\nThis integration collects metrics and logs from SonarQube\u2019s reports in order to provide insight into your codebase quality. You can monitor for code issues and security violations as they're reported by the scanner, visualize changes in code complexity, and collect logs from the SonarQube instance itself for unusual activity. For more information, check out [our documentation](https://docs.datadoghq.com/integrations/sonarqube/).",
+                "font_size": "12",
+                "has_padding": true,
+                "show_tick": false,
+                "text_align": "left",
+                "tick_edge": "left",
+                "tick_pos": "50%",
+                "type": "note",
+                "vertical_align": "center"
+            },
+            "id": 4764073356999656,
+            "layout": {
+                "height": 2,
+                "width": 12,
+                "x": 0,
+                "y": 0
+            }
+        },
```

Comment:
Need `overview` section next to `about` section. Both sections should be on the same row(s) to avoid `about` section being full-width.